### ISSUE TITLE
Parsing of @Success and @Failure comments is buggy

### ISSUE THREAD

**Issue by stefanMadzharov** [swaggo/swag#941]
**Describe the bug**
There is something wrong with the parsing of @Success and @Failure particularly of their comment fields when there aren't any `{param type}` and `data type` fields. Example:
`// @Failure 500 "Internal Server Error`
 If there are only numbers in the comment it can be arbitrarily long. Still when the comment is "Internal Server Error" `swag init` gives me an error saying: 
`ParseComment error in file placeholder/placeholder.go :can not parse response comment "500 "Internal Server Error""`

**Your swag version**
1.7.0

**Your go version**
1.15.7

**Desktop:**
 - OS: linux/amd64

**Comment by snaveen** [comment]
I am also facing this issue

**Comment by sammyne** [comment]
Same issue~

**Comment by stepotronic** [comment]
Looking at the code the format is not supported:
https://github.com/swaggo/swag/blob/17c1766b6349df2ab31d52e87be0ae3abca0d239/operation.go#L727
It needs to be 5 sections in total divided by spaces
`@Failure 500 {schemaType} refType responseDescription `

**Comment by belindaxinyi** [comment]
I am also facing this issue when trying to update the docs...

**Comment by ubogdan** [comment]
Hey guys, Is there anyone reading that stupid documentation? 

This is an invalid definition:
```
// @Failure 500 "Internal Server Error"
```
A working example can be found [here](https://github.com/swaggo/swag/blob/17c1766b6349df2ab31d52e87be0ae3abca0d239/example/celler/controller/examples.go#L27-L53).

if you need to add a "description" about why the error is happening this can be done as the [following](https://github.com/swaggo/swag/blob/17c1766b6349df2ab31d52e87be0ae3abca0d239/example/markdown/api/api.go#L36-L48) example.

Hope that was helpful.

**Comment by j-sv** [comment]
I'm facing the same issue on responses that don't contain a response body. From the Swagger docs:

> Empty Response Body
> Some responses, such as 204 No Content, have no body. To indicate the response body is empty, do not specify a content for the response:
> 
>           responses:
>             '204':
>               description: The resource was deleted successfully.

The examples given would set a type field on the response, which wouldn't be correct.

**Comment by ubogdan** [comment]
Is anyone open to write an implementation?

**Comment by j-sv** [comment]
@ubogdan, would you accept supporting it as `@Success 200 {null} null "An empty response"`.

**Comment by ubogdan** [comment]
I feel this definition may be more intuitive for a GO developer 
```
@Success 200 {object} nil "An empty response".
```

**Comment by ubogdan** [comment]
The feature was implemented and will be released in v1.7.2.
